toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -179,6 +179,26 @@ SystemConfig:
       Permissions: "664"
 ```
 
+## KernelCommandLine type
+
+Options for configuring the kernel.
```

Comment:
Are there no plans to set/override the kernel command line?

Reply:
Nope. Too difficult.

The complete set of args is split across multiple files:

- /boot/grub2/grub.cfg
- /boot/systemd.cfg
- /boot/mariner.cfg
- /boot/mariner-mshv.cfg
- /boot//grub2/grubenv

Trying to come up with an API to add/remove args from any of these files seems intractable. And worse, it is all subject to change (particularly in relation to introducing grub-mkconfig in Mariner 3.0).

So, I am just going to let the user use post-install scripts if they want to do more complex modifications.